src/Symfony/Component/Cache/Traits/ApcuTrait.php

```diff
@@ -54,7 +54,14 @@ protected function doFetch(array $ids)
         $unserializeCallbackHandler = ini_set('unserialize_callback_func', __CLASS__.'::handleUnserializeCallback');
         try {
             $values = [];
-            foreach (apcu_fetch($ids, $ok) ?: [] as $k => $v) {
+            $ids = array_flip($ids);
+            foreach (apcu_fetch(array_keys($ids)) ?: [] as $k => $v) {
+                if (!isset($ids[$k])) {
+                    // work around https://github.com/krakjoe/apcu/issues/247
```

Comment:
SymfonyInsight. Do we need to have here comments?

Reply:
I think so yes, I don't want to remove that line again by mistake